src/main/python/main.py

```diff
@@ -0,0 +1,1123 @@
+        elif type(node.value) is bool:
+            lit.setType(TypeParser.createFrom("bool", False))
+        elif type(node.value) is type(None):
+            lit.setType(TypeParser.createFrom("None", False))
+        elif type(node.value) is bytes:
+            lit.setType(TypeParser.createFrom("byte[]", False))
+        else:
+            debug_print(type(node.value))
+            raise NotImplementedError
+        return lit
+
+    def visit_FormattedValue(self, node):
+        debug_print(ast.dump(node))
+        raise NotImplementedError
+
+    def visit_JoinedStr(self, node):
+        debug_print(ast.dump(node))
+        raise NotImplementedError
+
+    def visit_List(self, node):
+        debug_print(ast.dump(node))
+
+        ile = InitializerListExpression()
+        self.add_loc_info(node, ile)
+
+        list = ArrayList()
+
+        for el in node.elts:
+            expr = self.visit(el)
+            list.add(expr)
+
+        ile.setInitializers(list)
+
+        return ile
+
+    def visit_Tuple(self, node):
+        # debug_print(ast.dump(node))
+        # lit = Literal()
+        # self.add_loc_info(node, lit)
+        # lit.setType(TypeParser.createFrom("Tuple", False))
+        # elts = []
+        # for e in node.elts:
+        #    elts.append(self.visit(e))
+        # lit.setValue(elts)
+        return Expression()
+
+    def visit_Set(self, node):
+        debug_print(ast.dump(node))
+        raise NotImplementedError
+
+    def visit_Dict(self, node):
+        debug_print(ast.dump(node))
+        # TODO implement it
+        # raise NotImplementedError
+        return Expression()
+
+    ### VARIABLES ###
+    def visit_Name(self, node):
+        debug_print(ast.dump(node))
+        ref = DeclaredReferenceExpression()
+        self.add_loc_info(node, ref)
+        ref.setName(node.id)
+
+        # resolved_ref = self.scopemanager.resolve(ref)
+        # if resolved_ref != None:
+        #    ref.setRefersTo(resolved_ref)
+        # else:
+        #    debug_print("Failed to resolve name.")
+        #    raise RuntimeError
+        return ref
+
+    def visit_Load(self, node):
+        debug_print(ast.dump(node))
+        raise NotImplementedError
+
+    def visit_Store(self, node):
+        debug_print(ast.dump(node))
+        raise NotImplementedError
+
+    def visit_Del(self, node):
+        debug_print(ast.dump(node))
+        raise NotImplementedError
+
+    def visit_Starred(self, node):
+        debug_print(ast.dump(node))
+        raise NotImplementedError
+
+    ### EXORESSIONS ###
+    def visit_Expr(self, node):
+        debug_print(ast.dump(node))
+        return self.visit(node.value)
+
+    def visit_UnaryOp(self, node):
+        debug_print(ast.dump(node))
+        unop = UnaryOperator()
+        self.add_loc_info(node, unop)
+        unop.setOperatorCode(node.op)
+        unop.setInput(self.visit(node.operand))
+        if isinstance(node.op, ast.UAdd):
+            unop.setName("UAdd")
```

Comment:
`operatorCode` and `name` should be set to the same value. Additionally, it should be set to the symbols, e.g. `+` instead of a name. There is some logic code that depends on that.